lib_eio_linux/eio_linux.ml

```diff
@@ -1101,14 +1107,22 @@ let flow fd =
       | `Send -> Unix.SHUTDOWN_SEND
       | `All -> Unix.SHUTDOWN_ALL
 
-    method unix_fd op = FD.to_unix op fd
-  end
+    method unix_fd (op: [`Peek | `Take]) = FD.to_unix op fd
```

Comment:
leftover ?

Reply:
This (explicit type annotation) was needed to silence compiler/dune warnings I believe.